README.md

```diff
@@ -8,3 +8,29 @@ An implementation of Circonus log-linear histograms.
     ./configure
     make
     make install
+
+## Tests
+
+    make tests
+
+## Documentation
+
+The circllhist API is documented in the header file [circllhist.h](src/circllhist.h).
+A html version can be generated using:
+
+    make docs
+
+This requires doxygen to be available on your system.
+
+## Python bindings
+
+Build and install the C library as described above.
+Then use the following command to install the python library.
+
+    cd src/python
```

Comment:
All the shell commands in the README should be consistently quoted as such. For example:
```
$ cd src/python
$ python setup.py install
```

Reply:
The quoting of this snippet is consistent with the ones found above.
Adding `$` at the beginning, will break copy-paste. So I'd rather not add it in.

Reply:
On second look at the README, all good. GitHub actually does the right thing here and interprets the indented lines as shell commands, no quoting markup necessary.